deploy/workloads/hello-world-localhost-vcs-metasim.yaml

```diff
@@ -0,0 +1,48 @@
+run_farm:
```

Comment:
I feel like there are enough of these to warrant a subdirectory in workloads or perhaps a different directory to put CI tests in, as I feel like it's getting a bit cluttered with a lot of similar config files.

Reply:
I agree. I wasn't sure if we wanted to use this PR to reorganize things. Happy to make a new directory and add things there (including the vitis CI configs). I was thinking it would be something like `deploy/workloads/ci/<subfolders here>`